### PR TITLE
canaries

### PR DESCRIPTION
**Pull request by jkminneti**

### CODE REVIEW

**Comment by jkminneti** [comment]
this is so broken and i dont know why WIP

File: cmd/fetch.go [hunk 1]
```diff
@@ -1,62 +0,0 @@
...
-)
-
-func init() {
-	rootCmd.AddCommand(fetchCmd)
-}
-
-var fetchCmd = &cobra.Command{
-	SilenceUsage: true,
-	Use:          "fetch [appName]",
-	Short:        "display all tuber yaml files",
```
Comment (jkminneti) [inline cmd/fetch.go:18]: tired of maintaining this. if you've used this command in the last 6 months i'll put it back

File: pkg/core/prerelease.go [hunk 2]
```diff
@@ -5,64 +5,47 @@ import (
...
 		`go-template="%s"`,
 		"{{range .status.containerStatuses}}{{.state.terminated.message}}{{end}}",
 	)
-	timeout := time.Now().Add(time.Minute * 10)
+	var timeout time.Time
+	if resourceTimeout == 0 {
+		timeout = time.Now().Add(time.Minute * 10)
+	} else {
+		timeout = time.Now().Add(resourceTimeout)
+	}
```
Comment (jaysonesmith) [inline pkg/core/prerelease.go:48]: What do you think about this style wise? Set the default straight off and only worry about changing it if we need.

```suggestion
	timeout time.Time := time.Now().Add(time.Minute * 10)
	if resourceTimeout > 0 {
	    timeout = time.Now().Add(resourceTimeout)
	}
```

File: pkg/core/prerelease.go [hunk 3]
```diff
@@ -40,10 +40,8 @@ func waitForPhase(name string, kind string, app *TuberApp, resourceTimeout time.
 		`go-template="%s"`,
 		"{{range .status.containerStatuses}}{{.state.terminated.message}}{{end}}",
 	)
-	var timeout time.Time
-	if resourceTimeout == 0 {
-		timeout = time.Now().Add(time.Minute * 10)
-	} else {
+	timeout := time.Now().Add(time.Minute * 10)
+	if resourceTimeout > 0 {
```
Comment (jaysonesmith) [inline pkg/core/prerelease.go]: 🎉

**Review by jaysonesmith — APPROVED** [review]

**Pull request merged**